### ISSUE TITLE
Feat: Lustre Site

### ISSUE THREAD

**Issue by versecafe** [lustre-labs/lustre#145]
### Lustre Framework Site
In the same vein as [nextjs.org](nextjs.org), [nuxt.com](nuxt.com), and [phoenixframework.org](phoenixframework.org) a site for the Lustre framework obviously built on Lustre, to provide a both information on the framework but also an example of it in use.

#### Very early small demo page
<img width="1325" alt="image" src="https://github.com/lustre-labs/lustre/assets/147033096/8a8d493c-e778-4827-8ae4-d2d2573aa93b">

Also lustre doesn't really have release logs so idk where I'd get the side data from so right now I'm just using lorem, also the whole UI is just a mock up more wondering if there's interest in the idea at all

**Comment by hayleigh-dot-dev** [comment]
Yeah! A site is on my long list of things to do 😔 We have `lustre.build` as a domain ready ^.^

**Comment by versecafe** [comment]
Is there any idea how hosting would be done since Gleam isn't exactly suitable for shoving on a Lamda and into Vercel for free hosting?

Also any idea on theme or styling to get something unique?

**Comment by hayleigh-dot-dev** [comment]
If we build a full stack thing (eg maybe to show off server components) then we can host on fly, if not we can host a static site on GitHub pages or netlify no problem.

I was just throwing together something generic, but anyone with a design eye would be very appreciated ^.^

https://github.com/lustre-labs/lustre/assets/9001354/e13ac491-cc43-4059-93ea-6de6e6a3b2c6

**Comment by versecafe** [comment]
I think it would be nice to show off Server components, it's one of the unique features that makes lustre more of a full framework then a UI library, I'd be happy to help with the UI side since I spend far more time on front end web work than I have with Gleam or really FP and MVC control as a whole

**Comment by hayleigh-dot-dev** [comment]
Not making any promises or commitments but I am very open to more suggestions / designs ^.^

**Comment by versecafe** [comment]
I'm thinking of trying to pull in some design elements around the beam VMs name just since it could give a fun design element with light rays, probably something around `pink-300` for tailwind colours to match with gleam as a base. The main thing I'm worried about is doing animations for interactivity maybe ffi to framer motion could solve it since that opens up a lot of options

**Comment by hayleigh-dot-dev** [comment]
That sounds like a fun idea!

**Comment by codeluggage** [comment]
Idea: Re-use the https://tour.gleam.run concept to turn the quickstart's into an interactive experience that can be followed along with and tried out right on the page. 

I'm not sure how feasible it is to have Lustre running in the page itself that way, but it would be a great experience and first impression!

**Comment by hayleigh-dot-dev** [comment]
Someone has actually been building that! https://gleamtours.com/lustre-tutorial/introduction/welcome-to-lustre/ very very cool.

**Comment by ashercn97** [comment]
obsessed omg
